Page Replacement Algorithms PR

shraiyya opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

PR FOR ISSUE #2054 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

**Please add DWOC label and level.**

shraiyya committed:
Merge pull request #8 from TesseractCoding/master

fork updation

shraiyya committed:
Create allocate.java

shraiyya committed:
Update allocate.java

shraiyya committed:
Update README.md

HarshCasper commented on Java/Operating-System/allocate.java:
```diff
@@ -0,0 +1,641 @@
+API
+Training
+Blog
+About
```
Please remove this. It looks copy-pasted

HarshCasper commented on Java/Operating-System/allocate.java:
```diff
@@ -0,0 +1,641 @@
+	int d;
+	node next;
+
+	node() //constructor of node class
```
Use comments above the Line of Code. Follow the same pattern across the rest of your code

HarshCasper reviewed (changes requested):
Few more changes in general: 

- Remove the extra lines of code which are redundant
- Put Comments above the Line of Code
- Run Code formatted to format the Code

shraiyya committed:
Did the mentioned changes

Put Comments above the Line of Code
Removed the extra lines of code that are redundant
Ran Code formatted

shraiyya commented:
I have done the changes.
Please review and add the tag of level of DWOC.
